akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/voice/UserJoinedVoiceConfEvtMsgHdlr.scala

```diff
@@ -5,6 +5,10 @@ import org.bigbluebutton.common2.msgs._
 import org.bigbluebutton.core.models.VoiceUsers
 import org.bigbluebutton.core.running.{ LiveMeeting, MeetingActor, OutMsgRouter }
 import org.bigbluebutton.core2.message.senders.MsgBuilder
+import org.bigbluebutton.core.models._
+import org.bigbluebutton.core2.message.senders.MsgBuilder
+import org.bigbluebutton.core.apps.users.UsersApp
+import org.bigbluebutton.core.models.UserLeftFlag

```

Comment:
already imported above via `models._`

Reply:
Same for `import org.bigbluebutton.core.models.VoiceUsers` - either keep the explicit imports or go back and remove the ones which are now imported twice